packages/tasks/src/tasks/video-text-to-text/about.md

```diff
@@ -0,0 +1,99 @@
+Most of the video language models can take in videos, multiple videos, images and multiple images. Some of these models can also take interleaved inputs, which can have images and videos inside the text, where you can refer to the input image and input video within the text prompt.
+
+## Different Types of Video Language Models
+
+Video language models come in three types:
+
+- **Base:** Pre-trained models that can be fine-tuned. 
+- **Instruction:** Base models fine-tuned on video-instruction pairs and answers. 
+- **Chatty/Conversational:** Base models fine-tuned on video conversation datasets.
+
+
+## Use Cases
+
+### Video Question Answering
+
+Video language models trained on video-question-answer pairs can be used for video question answering and generating captions for videos.
+
+### Video Chat
+
+Video language models can be used to have a dialogue about a video.
+
+### Video Recognition with Instructions
+
+Video language models can recognize images through descriptions. When given detailed descriptions of specific entities, it can classify the entities in a video.
+
+## Inference
+
+You can use the Transformers library to interact with video-language models. 
+Below we load [a video language model](https://huggingface.co/llava-hf/LLaVA-NeXT-Video-7B-hf), write a simple utility to sample videos, use chat template to format the text prompt, process the video and the text prompt and infer. 
+
+```python
+import uuid
+import requests
+import cv2
```

Comment:
maybe mention that you need to install opencv-python? The other dependencies are fine imo.